<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrackPopup</class>
 <widget class="QDialog" name="AddTrackPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Track</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="labelAlignment">
     <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="file_label">
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="choose_music_file_button">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Select Music File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>331</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="1" column="0">
     <widget class="QLabel" name="image_label">
      <property name="text">
       <string>Image:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="choose_image_file_button">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Select Image File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="title_label">
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="title_input"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="artist_label">
      <property name="text">
       <string>Artist</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="artist_input"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="album_label">
      <property name="text">
       <string>Album:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="album_input"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="date_label">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="tracklist_num_label">
      <property name="text">
       <string>Tracklist #:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="tracklist_num_input"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="image_path">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="date_input">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2025</year>
          <month>1</month>
          <day>2</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>9999</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>M/d/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2025</year>
          <month>1</month>
          <day>2</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_date_button">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>AddTrackPopup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>AddTrackPopup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
